src/components/organisms/SpeedDelivery/index.tsx

```diff
@@ -124,6 +148,8 @@ const index = () => {
           onClick={() => {
             setSpeed("Instant: 2-5 minutes");
             setFee("Transaction fees:0.001 BTC");
+            setFlag1(0);
+            handleClick1();
```

Comment:
this won't give a consistent behaviour as javascript is asynchronous. Anyway you don't need four states for background colour. Instead you can use the flag state.
 replace `backgroundColor: backgroundColorval` with `backgroundColor: flag1 === 0 ? "#F2F2F7" : "#FFFFFF"` and remove the states.